query/src/main/scala/filodb/query/exec/RangeVectorTransformer.scala

```diff
@@ -340,11 +340,9 @@ final case class RepeatTransformer(startMs: Long, stepMs: Long, endMs: Long, exe
             limit: Int,
             sourceSchema: ResultSchema,
             paramResponse: Seq[Observable[ScalarRangeVector]]): Observable[RepeatValueVector] = {
+    val rs = new RecordSchema(sourceSchema.columns, brSchema = sourceSchema.brSchemas)
     source.map { rv =>
-        RepeatValueVector(rv, startMs, stepMs, endMs,
-          new RecordSchema(sourceSchema.columns, brSchema = sourceSchema.brSchemas),
-          execPlan,
-          querySession.queryStats)
```

Comment:
quick question: Previously we were tracking the time taken to create the `RepeatValueVector` object using the `queryStats.CpuNanosCounter`. Wondering if we still need to do it or it will be tracked under the caller itself ?

Reply:
That was done when we iterated though the RV to get the first item, now that we just instantiate I removed that bit.